### PR TITLE
adding test cases for check-certificate-expiry.rb

### PR DESCRIPTION
**Pull request by zbintliff**

### CODE REVIEW

**Comment by zbintliff** [comment]
Tests that should have been added with #70 but I was lazy.

**Comment by eheydrick** [comment]
Awesome @zbintliff, :heart: the tests!

File: README.md [hunk 1]
```diff
@@ -11,6 +11,10 @@
 
 **check-autoscaling-cpucredits.rb**
 
+**check-beasntalk-elb-metric.rb**
```
Comment (analytically) [inline README.md]: typo

**Comment by analytically** [comment]
Can we merge this?

**Pull request merged**